glue.md

```diff
@@ -8,10 +8,10 @@
 
 The next step is to glue the central plexiglas box where the air is floating through. Try to keep the predrilled holes free of glue. It's recommended to place the glue on the 8 mm wide sides of the plexiglass.
 
-- Use the [arcylic glue](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part.)
+- Use the [glue (acrylic glue)](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part, the predrills must match.)
```

Comment:
```suggestion
- Use the [glue (acrylic glue)](tools.yml#acrifix_192){qty:some, Cat:tool}[i](glueingadvise.md) to combine the [predrilled 480mmx326mm plexiglass plate](fromstep){Qty:1} with two [predrilled 310x64 plexiglass plate](fromstep){Qty:2}, on the shorter edges. 8 mm should remain on both sides. (The side parts are put in front of the 3mm holes of the top part, the predrilled holes must match.)
```

Reply:
pilot holes seems to be the term in english. to use predrills was a bad decision